.github/workflows/integration-test-cli.yaml

```
@@ -0,0 +1,27 @@
+name: Integration Tests CLI
```

Should this be the same as the name of the job or do we want this to be different?
```suggestion
name: CLI Integration Tests
```

---

We can use the same name, I'll apply this suggestion.